packages/apps/shopify-api/lib/webhooks/index.ts

```diff
@@ -6,7 +6,79 @@ import {process} from './process';
 import {validateFactory} from './validate';
 import {HttpWebhookHandlerWithCallback, WebhookRegistry} from './types';
 
-export function shopifyWebhooks(config: ConfigInterface) {
+interface Webhooks {
+  /**
+   * Add shop-specific webhook handlers to the library registry,
+   * allowing you to register webhooks with Shopify and process HTTP webhook requests from Shopify.
+   * Unless your app needs different webhooks for different shops, we recommend using app-specific webhooks:
+   *
+   * {@link https://shopify.dev/docs/apps/build/webhooks/subscribe#app-specific-subscriptions}
+   *
+   * If you use only app-specific webhooks, you do not need to use `addHandlers`.
+   *
+   */
+  addHandlers: ReturnType<typeof addHandlers>;
+
+  /**
+   * Fetches the configured handlers for shop-specific webhooks for the given topic.
```

Comment:
I think the `getTopicsAdded` and `getHandlers` descriptions got swapped.

Reply:
Good catch, thanks! Fixed 👍